cmd/ci-secret-bootstrap/main.go

```diff
@@ -421,7 +438,8 @@ func main() {
 	})
 	defer logrus.Exit(0)
```

Comment:
Can I remove `defer logrus.Exit(0)`?

If we hit error later, fail the job, and send the alert.

Reply:
I guess? No idea what the purpose of it is

Reply:
This was added in 6c0e4425ec5566849c6cf3f86e46dedfe99cedf2 and is needed to ensure that logout always happens on process exit.

Reply:
(`defer` did not work since `logrus.Fatal` calls `os.Exit` and the `logrus` defer alone did not logout on successful runs)

Reply:
Is this the logic?
successful runs (no calls of `logrus.Fatal`), we use `logrus.Exit(0)` to activate `logout`.  

unsuccessful runs: `logrus.Fatal` is called which calls `os.Exit(1)` and `defer logrus.Exit(0)` wont be executed.

Reply:
Yes